dotnet/samples/KernelSyntaxExamples/Getting_Started/Step9_Safe_Chat_Prompts.cs

```diff
@@ -0,0 +1,235 @@
+        var kernelArguments = new KernelArguments()
+        {
+            ["input"] = "What is Seattle?",
+        };
+        var chatPrompt = @"
+            <message role=""user"">{{$input}}</message>
+        ";
+        WriteLine(await kernel.InvokePromptAsync(chatPrompt, kernelArguments));
+    }
+
+    private async Task ExampleEmptyInputVariableAsync(Kernel kernel)
+    {
+        var chatPrompt = @"
+            <message role=""user"">{{$input}}</message>
+        ";
+        WriteLine(await kernel.InvokePromptAsync(chatPrompt));
+    }
+
+    private async Task ExampleHtmlEncodedTextAsync(Kernel kernel)
+    {
+        string chatPrompt = @"
+            <message role=""user""><![CDATA[<b>What is Seattle?</b>]]></message>
+        ";
+        WriteLine(await kernel.InvokePromptAsync(chatPrompt));
+    }
+
+    private async Task ExampleCDataSectionAsync(Kernel kernel)
+    {
+        string chatPrompt = @"
+            <message role=""user""></message>
+        ";
+        WriteLine(await kernel.InvokePromptAsync(chatPrompt));
+    }
+
+    private async Task ExampleTextContentAsync(Kernel kernel)
+    {
+        var chatPrompt = @"
+            <message role=""user"">
+                <text>What is Seattle?</text>
+            </message>
+        ";
+        WriteLine(await kernel.InvokePromptAsync(chatPrompt));
+    }
+
+    private async Task ExamplePlainTextAsync(Kernel kernel)
+    {
+        string chatPrompt = @"
+            <message role=""user"">What is Seattle?</message>
+        ";
+        WriteLine(await kernel.InvokePromptAsync(chatPrompt));
+    }
+
+    private readonly IPromptTemplateFactory _promptTemplateFactory = new KernelPromptTemplateFactory();
+
+    private Task<string> RenderPromptAsync(string template, Kernel kernel, KernelArguments arguments)
+    {
+        return this.RenderPromptAsync(new PromptTemplateConfig
+        {
+            TemplateFormat = PromptTemplateConfig.SemanticKernelTemplateFormat,
+            Template = template
+        }, kernel, arguments);
+    }
+
+    private Task<string> RenderPromptAsync(PromptTemplateConfig promptConfig, Kernel kernel, KernelArguments arguments)
+    {
+        var promptTemplate = this._promptTemplateFactory.Create(promptConfig);
+        return promptTemplate.RenderAsync(kernel, arguments);
+    }
+
+    public Step9_Safe_Chat_Prompts(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    public class LoggingHandler : DelegatingHandler
+    {
+        private readonly ITestOutputHelper _output;
+
+        public LoggingHandler(HttpMessageHandler innerHandler, ITestOutputHelper output) : base(innerHandler)
+        {
+            _output = output;
+        }
+
+        protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+        {
+            // Log the request details
+            //this._output.WriteLine($"Sending HTTP request: {request.Method} {request.RequestUri}");
```

Comment:
Is this on purpose or just a demonstration that it's possible to output request details? Same in line 221 and 229.

Reply:
Removed the commented out code